src/settings_gui.cpp

```diff
@@ -291,11 +291,11 @@ struct GameOptionsWindow : Window {
 				break;
 
 			case WID_GO_BASE_GRF_DROPDOWN:
-				list = BuildSetDropDownList<BaseGraphics>(selected_index, (_game_mode == GM_MENU));
+				list = BuildSetDropDownList<BaseGraphics>(selected_index, true);
 				break;
 
 			case WID_GO_BASE_SFX_DROPDOWN:
-				list = BuildSetDropDownList<BaseSounds>(selected_index, (_game_mode == GM_MENU));
+				list = BuildSetDropDownList<BaseSounds>(selected_index, true);
```

Comment:
With this change all callers of `BuildSetDropDownList` pass `true`, so that parameter can be removed and `BuildSetDropDownList`'s internal logic can be simplified a bit.